### PR TITLE
add binding AWS SES

### LINKED ISSUE #1667: Update docs add new binding AWS SES
**What content needs to be created or modified?**
Add docs for new binding AWS SES 

**Describe the solution you'd like**

**Where should the new material be placed?**
https://docs.dapr.io/reference/components-reference/supported-bindings/

**The associated pull request from dapr/dapr, dapr/components-contrib, or other Dapr code repos**
- dapr: https://github.com/dapr/dapr/pull/3550
- componentes: https://github.com/dapr/components-contrib/issues/1019

**Additional context**

### PR DESCRIPTION
**Pull request by fjvela**
Thank you for helping make the Dapr documentation better!

**Please follow this checklist before submitting:**

- [ ] [Read the contribution guide](https://docs.dapr.io/contributing/contributing-docs/)
- [ ] Commands include options for Linux, MacOS, and Windows within codetabs
- [ ] New file and folder names are globally unique
- [ ] Page references use shortcodes instead of markdown or URL links
- [ ] Images use HTML style and have alternative text
- [ ] Places where multiple code/command options are given have codetabs

In addition, please fill out the following to help reviewers understand this pull request:

## Description
 New binding AWS SES

## Issue reference
https://github.com/dapr/docs/issues/1667

### CODE REVIEW

**Comment by AaronCrawfis** [comment]
Waiting on https://github.com/dapr/components-contrib/issues/1019

**Comment by orizohar** [comment]
This PR should target the pre-release branch which is now v1.4. Thanks

File: daprdocs/content/en/reference/components-reference/supported-bindings/ses.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+The `emailTo`, `emailCC` and `emailBCC` fields can contain multiple email addresses separated by a semicolon.
+
+Example:
+```json
+{
+  "operation": "create",
+  "metadata": {
+    "emailTo": "dapr-smtp-binding@example.net",
+    "emailCC": "cc1@example.net",
```
Comment (AaronCrawfis) [inline daprdocs/content/en/reference/components-reference/supported-bindings/ses.md]: Does case sensitivity matter here? Just curious as emailCC is listed here, but emailCc is listed in the table above.

Comment (fjvela) [reply daprdocs/content/en/reference/components-reference/supported-bindings/ses.md]: Fixed.

**Review by orizohar — APPROVED** [review]

**Pull request merged**